src/pages/ResourceForm.tsx

```diff
@@ -61,12 +63,18 @@ interface ResourceFormProps {
   resourceType: ResourceCreationType;
 }
 
+const fileUploadPanelId = 'file-upload-panel';
+
 const ResourceForm: FC<ResourceFormProps> = ({ uppy, resource, resourceType }) => {
   const { t } = useTranslation();
   const [allChangesSaved, setAllChangesSaved] = useState(true);
   const [isLoadingLicenses, setIsLoadingLicenses] = useState(false);
   const [loadingLicensesErrorStatus, setLoadingLicensesErrorStatus] = useState(StatusCode.ACCEPTED); //todo: String
   const [licenses, setLicenses] = useState<License[]>();
+  const [percentageFileUpload, setPersentageFileUpload] = useState(0);
+  const [count, setCount] = useState<number>(0);
```

Comment:
count brukes  ikke ?

Reply:
count brukes her:
linje 155
useInterval(() => {
    setPersentageFileUpload(uppy.getState().totalProgress);
    setCount(count + 1);
  }, calculateShouldUseInterval());

Reply:
vel den setter jo bare seg selv. Er det for å tvinge fram en ny render av komponenten ? i såfall er ikke logikken tydelig

Reply:
Fjernet